pxr/usd/usdMtlx/parser.cpp

```diff
@@ -48,6 +48,26 @@ TF_DEFINE_PRIVATE_TOKENS(
 
     ((discoveryType, "mtlx"))
     ((sourceType, ""))
+
+    (colorspace)
+    (defaultgeomprop)
+    (defaultinput)
+    (doc)
+    (enum)
```

Comment:
@JGamache-autodesk , did this compile for you?  `enum` is not a legal token/identifier name in C++.  YOu can do something like:
```
((enum_, "enum"))
```

Thanks!

Reply:
@spiffmon I added the patch (from Jerry).

Reply:
Looks great - thanks, @seando-adsk !